app/__tests__/services/signaling.test.ts

```
@@ -0,0 +1,670 @@
+
+// Mock for Date.now
+const mockTimestamp = 1234567890;
+const originalDateNow = Date.now;
+Date.now = jest.fn(() => mockTimestamp);
+
+/**
+ * Mock implementation of ApiInterface for testing
+ */
+class MockApiClient implements ApiInterface {
+  private messageQueue: SignalingMessage[] = [];
+  private mockUserId: string = 'test-user-id';
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)